Content.Shared/ReagentOnItem/Components/SpaceGlueOnItemComponent.cs

```diff
@@ -0,0 +1,18 @@
+namespace Content.Shared.ReagentOnItem;
+
+[RegisterComponent, AutoGenerateComponentPause]
+public sealed partial class SpaceGlueOnItemComponent : ReagentOnItemComponent
+{
+    /// <summary>
+    ///     The time you should check if the glue component has run out.
+    /// </summary>
+    [DataField, AutoPausedField]
+    public TimeSpan TimeOfNextCheck;
+
+    /// <summary>
+    ///     How long the item will be stuck to someones hand per unit
+    ///     of glue.
```

Comment:
```suggestion
    ///     How long the item will be stuck to someones hand per unit of glue.
```
Your editor should have the line break set at 120.